packages/forms/stories/Playground.stories.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { action } from '@storybook/addon-actions';
 import isEqual from 'lodash/isEqual';
 
-import { FormDefinition } from '../lib';
+import { FormDefinition } from '../src/types';
```

Comment:
if you import from lib you have to build the lib to be able to build the story. so it fails